source/_integrations/drop_connect.markdown

```diff
@@ -37,6 +37,7 @@ There is currently support for the following DROP products within Home Assistant
 - **Pump Controller**: smart replacement for an FSG pressure switch.
 - **RO Filter**: reverse osmosis drinking water filtration.
 - **Salt Sensor**: alerts when the salt level in the softener brine tank is low.
+- **Alert**: monitors an electrical circuit and a float or dry contact input, ideal for monitoring a sump pump.
```

Comment:
```suggestion
- **Alert**: monitors an electrical circuit and a float or dry contact input. It detects high or low water levels, monitors the temperature, and provides an audible alarm. This makes it ideal for monitoring devices such as a sump pump used to keep the basement dry.
```
Added a bit more context to make it easier to understand what this does. Feel free to edit.

Reply:
I think my preference would be to leave the description less verbose, simply because all of the other product descriptions on the list are intentionally very short.  Expounding on just this one product feels out of place.  I wanted to have a list of supported devices in this documentation without it sounding like a sales pitch.

I'm open to adding more detail about each of the products on the list to match the level of detail in your suggestion, but I'm assuming that should be done as a separate pull request because it's outside the scope of adding just this one product.  Thoughts?